### DISCUSSION TITLE
switching google play store accounts

### DISCUSSION THREAD

**Discussion by ILikeBypassing** [3kh0/ext-remover#253] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
the administator disabled switching accounts on the google play store to prevent us from download other "non-educational" apps. so if there even is a way to switch accounts even when its disabled please tell me.

**Comment by Idunnoman001** [comment, marked as answer]
you can downgrade your chromebook but i dont recommend it because you can get in serious trouble if the school finds out.
